.github/workflow/tests.yml

```diff
@@ -0,0 +1,72 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lsp-tests:
+    name: LSP tests - java v${{ matrix.java_version }} & ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    needs: test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup go 1.13.1
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.1
+
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: '12'
+
+      - uses: bugdiver/setup-gauge@master
```

Comment:
Should be move this repo to getgauge?

Reply:
cc @BugDiver

Reply:
Sure We can do that.

Reply:
Have transferred the repo. Please update the links